docs/backends/lua2.rst

```diff
@@ -2,13 +2,18 @@ Lua2 Backend
 ============
 
 * Native: Yes
-* Master: Yes
-* Slave: No
-* Superslave: No
+* Primary: Yes
+* Secondary: No
+* Producer: No
+* Consumer: No
+* Autosecondary: No
+* DNS Update: No
 * DNSSEC: Yes
 * Disabled data: No
 * Comments: No
-* Zone caching: Yes
+* API: Read-Write
+* Multiple instances: Yes
+* Zone caching: No
```

Comment:
Yes - if `dns_get_all_domains` is implemented

Reply:
which I guess is just Yes, we shouldn't have to note "if implemented" on every item here